src/platform/android/java/chip/platform/NsdServiceFinderAndResolver.java

```diff
@@ -159,9 +160,10 @@ public void onServiceResolved(NsdServiceInfo serviceInfo) {
                     + "' to "
```

Comment:
Can you also add the serviceType in the logs here too? That way it might be useful understanding when it gets stripped

Reply:
Added. Thanks.